pkgs/applications/window-managers/sway/idle.nix

```diff
@@ -22,7 +22,7 @@ stdenv.mkDerivation rec {
 
   mesonFlags = [ "-Dman-pages=enabled" "-Dlogind=${if systemdSupport then "enabled" else "disabled"}" ];
 
-  # Remove the `%zu` patch for the next release after 1.7.1.
+  # Remove the `%zu` patch for the next release after 1.8.0.
   # https://github.com/swaywm/swayidle/commit/e81d40fca7533f73319e76e42fa9694b21cc9e6e
```

Comment:
This substitution should be dropped completely as the linked commit is in this release. (Sorry, GitHub won't let me suggest deleting the lines outside the diff)

Reply:
Thanks :) And sorry for the long delay - I finally got around to handling this PR.